requirements.txt

```diff
@@ -18,18 +19,23 @@ attrs==23.1.0
 Babel==2.13.1
 backcall==0.2.0
 beautifulsoup4==4.12.2
+black==24.4.2
```

Comment:
why was  this added in?

Reply:
Was doing some experiments with some other libraries. Looking into it.

Reply:
Looks like it was added by `datamodel-code-generator`, but I'm not using that anymore. Gonna delete and see if there are any issues.

Reply:
I was able to remove the dependencies without issue. Gonna resolve this issue for now.